pkg/api/handlers/libpod/containers.go

```diff
@@ -325,3 +330,129 @@ func ListContainerBatch(rt *libpod.Runtime, ctr *libpod.Container, opts shared.P
 	}
 	return ps, nil
 }
+
+func Checkpoint(w http.ResponseWriter, r *http.Request) {
```

Comment:
Needs doc comment

Reply:
Can wait till later, though, given this is just an HTTP endpoint

Reply:
none of the endpoints have doc comments